includes/admin/views/html-admin-page-status.php

```diff
@@ -35,7 +35,11 @@
 				WC_Admin_Status::status_logs();
 			break;
 			default :
-				WC_Admin_Status::status_report();
+				if ( array_key_exists( $current_tab, $tabs ) && has_action( 'woocommerce_admin_status_' . $current_tab ) ) {
```

Comment:
Fixed in https://github.com/woothemes/woocommerce/pull/10874/commits/98a15d52d7a5ebb32a425b659b7dde41791c8e0a